config/config_mgmt.py

```diff
@@ -111,6 +113,136 @@ def readConfigDB(self):
 
         return
 
+    def writeConfigDB(self, jDiff):
+        print('Writing in Config DB')
+        """
+        On Sonic Switch
+        """
+        db_kwargs = dict(); data = dict()
+        configdb = ConfigDBConnector(**db_kwargs)
+        configdb.connect(False)
+        deep_update(data, FormatConverter.to_deserialized(jDiff))
+        data = sort_data(data)
+        self.logInFile("Write in DB: Last Data\n", data)
+        configdb.mod_config(FormatConverter.output_to_db(data))
+
+        return
+
+    """
+     Check ASIC DB whether interfaces present or not
+     ports: List of ports
+     portMap: port to OID map.
+     Return: True if all ports are not present.
```

Comment:
You mean no port are present? Maybe change the name to checkNoPortsInAsicDb() to remove confusion.

Reply:
I will make a function to check one port`s presence, after that I will check for False from this function.